plugins/woocommerce/src/Internal/Admin/RemoteFreeExtensions/DefaultFreeExtensions.php

```diff
@@ -824,47 +824,72 @@ public static function get_plugin( $slug ) {
 	public static function with_core_profiler_fields( array $plugins ) {
```

Comment:
Perhaps we should comment about how how `install_priority` is handled here

Reply:
Good idea. I'll copy the table from this PR to the comment.

Reply:
Updated in https://github.com/woocommerce/woocommerce/pull/38616/commits/31f8600ae5bc465e5481b836fe9b1ec2bbc8a3fb